bot/moderation/moderation.py

```diff
@@ -24,10 +25,48 @@ def __init__(self, bot):
         self.bot = bot
         self._db_connector = DatabaseConnector(constants.DB_FILE_PATH, constants.DB_INIT_SCRIPT)
 
-    @commands.group(invoke_without_command=True)
+    @commands.command(name='report')
     @command_log
-    async def modmail(self, ctx: commands.Context):
+    async def report_user(self, ctx: commands.Context, offender: discord.Member, *, description: str):
+        """Command Handler for the `report` command.
+
+        Allows users to report other members to the moderators by using their ID, name + discriminator, name, nickname
+        or by simply mentioning them. Everything after that will be used as the description for the report. The
+        complete report will then be posted in the configured report channel, which (hopefully) can only be accessed by
+        by the moderators.
+        If no member could be found, the user who invoked the command will be informed via a direct message.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            offender (discord.Member): The person accused of doing something wrong.
+            description (str): A description of why this person has been reported.
+        """
+        await ctx.message.delete()
 
+        embed = _create_report_embed(offender, ctx.author, ctx.channel, ctx.message, description)
+        await ctx.guild.get_channel(constants.CHANNEL_ID_REPORT).send(embed=embed)
+
+    @report_user.error
+    async def report_error(self, ctx: commands.Context, error: commands.CommandError):
+        """Error Handler for the `report` command.
+
+        Handles specific exceptions which occur during the execution of this command. The global error handler is still
+        being called for every error thrown.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context in which the command was called.
+            error (commands.CommandError): The error raised during the execution of the command.
+        """
+        if isinstance(error, commands.BadArgument):
+            regex = re.search("\"(.*)\"", error.args[0])
```

Comment:
What does this regex search for (hard to read in code)? maybe a short inline comment?

Reply:
Get the text between two quotes.